src/main/java/io/cryostat/net/reports/AbstractReportGeneratorService.java

```diff
@@ -65,18 +63,12 @@
 
     protected final TargetConnectionManager targetConnectionManager;
     protected final FileSystem fs;
-    // FIXME extract TempFileProvider to FileSystem
```

Comment:
Does this PR also fix #567 as well? If so, could we also remove the `tempFileProvider` object at https://github.com/cryostatio/cryostat/blob/main/src/test/java/io/cryostat/net/reports/ActiveRecordingReportCacheTest.java#L79 ? It looks like it's unused.

Reply:
Ah right, that's where the issue for it was. I thought it was filed in -core but couldn't find it - no wonder why.